charts/federation-v2/values.yaml

```diff
@@ -18,6 +18,20 @@ controllermanager:
     requests:
       cpu: 100m
       memory: 64Mi
+  ## Feature gates are a set of key=value pairs that describe alpha or
+  ## experimental features. An administrator can use the --feature-gates
+  ## command line flag on each component to turn a feature on or off.
+  ## Federation V2 now have the following default feature gates:
+  ## PushReconciler=true,SchedulerPreferences=true,CrossClusterServiceDiscovery=true,FederatedIngress=true
```

Comment:
(No action required) Ugh, this is gross.  I think we need to transition to versioned component config to ensure that federation config can be backwards compatible or risk complicating upgrades.

cc: @pmorie